pages/verification/[slug]/index.tsx

```diff
@@ -24,14 +37,63 @@ const VerificationRoute = () => {
 		};
 	}, []);
 
-	if (isLoading) {
+	useEffect(() => {
+		setIsProjectLoading(true);
+		setAllowVerification(false);
+		setOwnerAddress(undefined);
+		setIsCancelled(false);
+		client
+			.query({
+				query: FETCH_PROJECT_BY_SLUG,
+				variables: {
+					slug,
+					connectedWalletUserId: Number(userData?.id),
+				},
+				fetchPolicy: 'network-only',
+			})
+			.then((res: { data: { projectBySlug: IProject } }) => {
+				const _project = res.data.projectBySlug;
+				const isOwner = compareAddresses(
+					userData?.walletAddress,
+					_project.adminUser.walletAddress,
+				);
+				setOwnerAddress(_project.adminUser.walletAddress);
+				if (_project.status.name === EProjectStatus.ACTIVE && isOwner) {
```

Comment:
setAllowVerification(_project.status.name === EProjectStatus.ACTIVE && isOwner);

Reply:
Making it concise helps us maintain it better.

Reply:
done!